.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: 🐞 Bug Report
+about: Report a reproducible bug or issue
+title: "[Bug]: <describe bug>"
+labels: bug
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Run command '...'
+3. See error
+
+**Expected behavior**
+What you expected to happen.
+
+**Environment (please complete the following):**
+ - OS: [e.g. Ubuntu 22.04]
+ - kubectl-ai version: [e.g. 0.3.0]
```

Comment:
In parenthesis, can we add quick command on how to get version. `kubectl-ai version`